### PR TITLE
[SERVICE-278] Prevent resizing of the top edge of a tabbed window

### PR DESCRIPTION
**Pull request by tomrobertsOF**
Changed some of the constraints propagation logic to restrict resizing on the internal edge of a tabgroup.

### CODE REVIEW

File: test/demo/tabbing/tabGroupResizeConstraints.test.ts [hunk 1]
```diff
@@ -154,7 +154,7 @@ function constraintsUnion(...windowConstraints: Constraints[]): Required<Constra
 
     // Check if the union of side constraints made the window non-resizable
     if (result.resizable) {
-        result.resizable = (Object.keys(result.resizeRegion.sides) as Side[]).every(key => result.resizeRegion.sides[key] !== false);
+        result.resizable = (Object.keys(result.resizeRegion.sides) as Side[]).some(key => result.resizeRegion.sides[key] !== false);
```
Comment (bryangaleOF) [inline test/demo/tabbing/tabGroupResizeConstraints.test.ts]: Would be less elegant, but could be worth defining resizable as 'resizable left, right, and at least one of top and bottom', and non-resizable as 'non-resizable on all edges', and throwing an error for anything that doesn't fit one of those definitions (or would we sometimes expect windows to not fit those definitions?)

Comment (tomrobertsOF) [reply test/demo/tabbing/tabGroupResizeConstraints.test.ts]: As discussed, this is intended.

**Comment by tomrobertsOF** [comment]
Just waiting for Jenkins before merging.

**Comment by tomrobertsOF** [comment]
CI build has passed. Will merge in a few minutes if no other objections

**Pull request merged**